.github/CODEOWNERS

```diff
@@ -1939,3 +1939,4 @@ spec/lib/va_profile/profile/v3/service_spec.rb @department-of-veterans-affairs/v
 lib/logging/third_party_transaction.rb @department-of-veterans-affairs/backend-review-group
 spec/lib/logging/third_party_transaction_spec.rb @department-of-veterans-affairs/backend-review-group
 spec/requests/va_profile/contacts_request_spec.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+/helmCharts/ @department-of-veterans-affairs/backend-review-group
```

Comment:
Missing final new line. Also, I don't th ink we want to start with a slash. Not sure though.

Reply:
adding the new line - kept the leading / as it denotes in the root of the repository. The trailing slash denotes a directory - but removing since we have the leading slash